Dockerfiles/Dockerfile.jenkins

```diff
@@ -0,0 +1,20 @@
+FROM python:3.7.10
+# For build CBC Jenkins job ECR image
+ENV PYTHONUNBUFFERED 1
+# ENV PYTHONDEVMODE 1
+
+RUN mkdir /code
+ADD . /code/
+WORKDIR /code
+
+RUN apt-get update && apt-get install -yq git unzip curl
+
+# Install Chrome for Selenium
+RUN curl https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb -o /chrome.deb \
+    && dpkg -i /chrome.deb || apt-get install -yf \
+    && rm /chrome.deb
+
+# Install chromedriver for Selenium
+RUN wget -O /tmp/chromedriver.zip http://chromedriver.storage.googleapis.com/`curl -sS chromedriver.storage.googleapis.com/LATEST_RELEASE`/chromedriver_linux64.zip \
+    && unzip /tmp/chromedriver.zip chromedriver -d /usr/local/bin/ \
+    && chmod +x /usr/local/bin/chromedriver

```

Comment:
Can we rename this file to Dockerfile.selenium-jenkins just to clarify that its specific to selenium testing within Jenkins?